src/test/java/net/sf/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -76,6 +76,14 @@ public void findByEprint() throws IOException {
     }
 
     @Test
+    // Test for https://github.com/JabRef/jabref/issues/2427
```

Comment:
I would omit the comment here. It is a test for expected behavior. Doesnt matter which bug/problem it fixed.